natspec/interfaces/IAllowlistV1.md

```diff
@@ -13,7 +13,7 @@ This interface exposes methods to handle the list of allowed recipients.
 ### allow
 
 ```solidity
-function allow(address[] _accounts, uint256[] _permissions) external nonpayable
+function allow(address[] _accounts) external nonpayable
```

Comment:
is this autogenerated ? I guess this needs to be re-run if so ?

Reply:
Generated again